processor.go

```diff
@@ -607,7 +607,11 @@ func (g *Processor) createPartitionProcessor(ctx context.Context, partition int3
 		return fmt.Errorf("processor [%s]: partition %d already exists", g.graph.Group(), partition)
 	}
 
-	pproc := newPartitionProcessor(partition, g.graph, session, g.log, g.opts, g.lookupTables, g.saramaConsumer, g.producer, g.tmgr)
+	backoff, err := g.opts.builders.backoff()
```

Comment:
this could be done in the partition_processor as it has the options anyway.

Reply:
As said above. Since this function retrunes an error this would mean altering the newPartitionProcessor signature.

Reply:
true